### PR TITLE
Add system tests

### PR DESCRIPTION
**Pull request by mcass19**

### CODE REVIEW

File: spec/system/new_retrospective_spec.rb [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'NewRetrospective' do
+  before do
+    driven_by :chrome
```
Comment (rodrigopdl) [inline spec/system/new_retrospective_spec.rb]: Are all the `before` blocks to set the `driven_by :chrome` setting necessary? 
Or can this be configured for all tests in a config file?

File: app/views/retrospectives/_action_plans_item.html.erb [hunk 2]
```diff
@@ -1,3 +1,3 @@
-<div  class='topics list plan'> 
+<li class='topics list plan'>
```
Comment (rodrigopdl) [inline app/views/retrospectives/_action_plans_item.html.erb]: What's the reason for moving from `div`'s to `li`'s?

File: spec/system/home_page_spec.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+require 'rails_helper'
+
+describe 'HomePage' do
+  before do
+    driven_by :chrome
+  end
+
+  it 'renders the image, text and button of the home page' do
+    visit root_url
+    expect(page).to have_css "img[src*='wye-logo']"
```
Comment (rodrigopdl) [inline spec/system/home_page_spec.rb]: I sometimes like to add some blank lines as a means to separate different sections in a spec, all with the goal of making it a bit more readable. For example, I generally try to keep `expect`ations separate from the test variables set-up or commands.

For this specific case, I might do something like this:

```ruby
it 'renders the image, text and button of the home page' do
    visit root_url

    expect(page).to have_css "img[src*='wye-logo']"
    expect(page).to have_text 'Welcome to WyeRetro App!'
    expect(page).to have_button 'Start'

    click_button 'Start'

    expect(page).to have_current_path(retrospectives_path)
  end
```

It's a personal preference. What are your thoughts on this?

File: app/views/retrospectives/_action_plans_item.html.erb (hunk 2 above)
Comment (mcass19) [reply app/views/retrospectives/_action_plans_item.html.erb]: The reason was to be able to sort the elements in the test, for example, sort `#mention_list li: first-child`.

Also, it's a list really, so I think it was not a bad decision, but if you think that the previous way was fine, I turn around the changes

File: spec/system/new_retrospective_spec.rb (hunk 1 above)
Comment (mcass19) [reply spec/system/new_retrospective_spec.rb]: I think I forgot to change that 😅.

I made those changes in 7578a51.

File: spec/system/home_page_spec.rb (hunk 3 above)
Comment (mcass19) [reply spec/system/home_page_spec.rb]: I totally agree with you! It is much more readable by writing it this way. Thank you !

I changed it in a31ce4e.

File: spec/spec_helper.rb [hunk 4]
```diff
@@ -16,6 +21,14 @@
 # it.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+
+Capybara.register_driver(:chrome) do |app|
```
[the remark this answers is not in the document]
Comment (mcass19) [reply spec/spec_helper.rb]: Great ! I've removed those settings and everything goes well!
I did it in 9f1d272.

**Pull request closed without merging**